src/AstTranslator.cpp

```diff
@@ -1214,8 +1235,8 @@ std::unique_ptr<RamStatement> AstTranslator::makeSubproofSubroutine(const AstCla
 
 /** make a subroutine to search for subproofs for the non-existence of a tuple */
 std::unique_ptr<RamStatement> AstTranslator::makeNegationSubproofSubroutine(const AstClause& clause) {
-    // TODO (taipan-snake): Currently we only deal with atoms (no constraints or negations or aggregates or
-    // anything else...)
+    // TODO (taipan-snake): Currently we only deal with atoms (no constraints or negations or aggregates
```

Comment:
Why the reflowing of all these comments? I'm not seeing a change, apart from reflowing and adding to the diff.

Reply:
This is my fault by not using clang-format properly.